mycroft/skills/phillips_hue/vocab/en-us/ActivateSceneKeyword.voc

```diff
@@ -0,0 +1,6 @@
+activate scene
+activate seen
+set my lights to
+set scene
+set seen
```

Comment:
If we the best we can do is 'control them more,' I'd still argue for keeping the alternate spellings. Developing and watching the logs, I can see and understand why Mycroft didn't understand my query - he got 'seen' instead of 'scene.' To an end user, however, when Mycroft keeps saying "I am searching for activate seen reading," they're going hear 'seen,' be unable to differentiate it from 'scene,' and assume he just took the wrong action, which is bad.

I'll admit that I don't know how the STT works, so correct me if I'm way off base here, but I feel like altering the phrasing would just be crossing my fingers that it's interpreted more accurately, and that doesn't seem good enough. I'll also admit it's a hack to have the alternate spellings, but it's a hack we know works 100% of the time.